### PR TITLE
law of space - lines

### PR DESCRIPTION
**Pull request by Lellahahaha**
Awesome you want to make a PR  
please see [https://github.com/fabianmoronzirfas/gestalten-in-code/blob/master/.github/CONTRIBUTING.md](https://github.com/fabianmoronzirfas/gestalten-in-code/blob/master/.github/CONTRIBUTING.md) for further infos.

### CODE REVIEW

File: form/2d/p5js/index.md [hunk 1]
```diff
@@ -0,0 +1,9 @@
+---  
+layout: p5js
+author: Jeanette Knipp
+author-url: https://github.com/Lellahahaha
+title: An Obvious Circle
+tags: [P5.js, Form, Law of Closure]
+permalink: https://github.com/fabianmoronzirfas/gestalten-in-code/tree/master/form
```
Comment (fabianmoronzirfas) [inline form/2d/p5js/index.md]: The permalink should be the category and your title e.g. 

    form/an-obvious-circle/

File: form/3d/index.md [hunk 2]
```diff
@@ -0,0 +1,9 @@
+---  
+layout: p5js
+author: Jeanette Knipp
+author-url: https://github.com/Lellahahaha
+title: Lines for Space
+tags: [P5.js, Form, Law of Space]
+permalink: https://github.com/fabianmoronzirfas/gestalten-in-code/tree/master/form
```
Comment (fabianmoronzirfas) [inline form/3d/index.md]: The permalink should be the category and your title e.g. 

    form/lines-for-space/

**Comment by Lellahahaha** [comment]
Done!

File: form/2d/p5js/An Obvious Circle/index.md [hunk 3]
```diff
@@ -0,0 +1,9 @@
+---  
+layout: p5js
+author: Jeanette Knipp
+author-url: https://github.com/Lellahahaha
+title: An Obvious Circle
+tags: [P5.js, Form, Law of Closure]
+permalink: form/An Obvious Circle
```
Comment (fabianmoronzirfas) [inline form/2d/p5js/An Obvious Circle/index.md]: the folder names and permalinks need to be URL save so:  

    form/An Obvious Circle  

should be:  

    /form/an-obvious-circle/

File: form/3d/p5js/Lines for Space/index.md [hunk 4]
```diff
@@ -0,0 +1,9 @@
+---  
+layout: p5js
+author: Jeanette Knipp
+author-url: https://github.com/Lellahahaha
+title: Lines for Space
+tags: [P5.js, Form, Law of Space]
+permalink: form/Lines for Space
```
Comment (fabianmoronzirfas) [inline form/3d/p5js/Lines for Space/index.md]: This permalink and the folder name need to be URL save. Please change it to

    /form/lines-for-space/

File: form/2d/p5js/An Obvious Circle/index.js [hunk 5]
```diff
@@ -0,0 +1,31 @@
+function setup() {
+  createCanvas(400,400);
```
Comment (fabianmoronzirfas) [inline form/2d/p5js/An Obvious Circle/index.js]: The canvas need to be assigned to div. Please change the code to this:  

```
  var canvas = createCanvas(500, 500); // !this is important
  canvas.parent('sketch'); // !this as well
```

File: form/3d/p5js/Lines for Space/index.js [hunk 6]
```diff
@@ -0,0 +1,8 @@
+function setup() {
+var canvas = createCanvas (400, 400);
```
Comment (fabianmoronzirfas) [inline form/3d/p5js/Lines for Space/index.js]: Please make the canvas 500 × 500 px

Comment (Lellahahaha) [inline form/3d/p5js/Lines for Space/index.js]: okay, done!

**Comment by fabianmoronzirfas** [comment]
Hm. You still did not change the folder names. I will merge this and change it myself for the sake of writing to many comments.

**Pull request merged**